src/submission/validation-clinical/treatment.ts

```diff
@@ -157,3 +174,27 @@ function getTreatment(
     clinicalInfo: { [idFieldName]: treatmentId as string },
   }) as DeepReadonly<Treatment>;
 }
+
+function checkFollowUpTimeConflict(
```

Comment:
can we generalize this function in utils since it's used in two places ?  it's better to avoid repeating code